sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client_operations.cpp

```diff
@@ -149,7 +149,7 @@ Azure::Response<DeletedCertificate> DeleteCertificateOperation::PollUntilDoneInt
 std::unique_ptr<Azure::Core::Http::RawResponse> DeleteCertificateOperation::PollInternal(
     Azure::Core::Context const& context)
 {
-  std::unique_ptr<Azure::Core::Http::RawResponse> rawResponse;
+  std::unique_ptr<Azure::Core::Http::RawResponse> rawResponse = std::move(m_rawResponse);
```

Comment:
Same comment as previous - this seems an unexpected mechanism to free `m_rawResponse`.